packages/discovery-provider/src/queries/get_attestation.py

```diff
@@ -193,8 +194,8 @@ def get_attestation(
 
     attestation = Attestation(
         amount=str(user_challenge.amount),
-        oracle_address=oracle_address,
-        user_address=user_address,
+        oracle_address=oracle_address.lower(),
```

Comment:
should we test this behavior
https://github.com/AudiusProject/audius-protocol/blob/44b99b611d0df322ba95ef90c41cdbdc72b86f89/packages/discovery-provider/integration_tests/queries/test_get_attestation.py#L133

Reply:
removed the behavior instead, opting to have sdk do the work